project/Dependencies.scala

```diff
@@ -171,10 +173,12 @@ object Dependencies {
     commonsCodec,
     liquibaseCore,
     circeYAML,
+    snakeYAML,
```

Comment:
I forgot to add tis to the list last pr -- i actually did verify that the snakeYAML version already got bumped with circe but this is more explicit

Reply:
I usually prefer these be in separate PRs for sanity and causation reasons if something breaks, but not worth making another PR for it now.  In the future though, I'd say try to keep them separate even though it's annoying.